kubernetes/crd/dataset.json

```diff
@@ -0,0 +1,35 @@
+{
+	"apiVersion": "apiextensions.k8s.io/v1",
+	"kind": "CustomResourceDefinition",
+	"metadata": { "name": "datasets.new.artificialwisdom.cloud" },
+	"spec": {
+		"group": "new.artificialwisdom.cloud",
+		"versions": [{
+			"name": "v1",
+			"served": true,
+			"storage": true,
+			"schema": {
+				"openAPIV3Schema": {
+					"type": "object",
+					"properties": {
+						"spec": {
+							"type": "object",
+							"properties": {
+								"source": { "type": "string" },
+								"filename": { "type": "string" },
```

Comment:
if filename an URI within our cloud, or something else? I don't know much about `doi`, although it may be interesting to add and automate, if it is useful. Perhaps file a tracking bug for the idea?

Reply:
This is just a filename into an OCI bucket. We could always generate them systematically.